src/core/components/AMInstallButton/index.js

```diff
@@ -354,9 +338,6 @@ export class AMInstallButtonBase extends React.Component<InternalProps> {
           buttonProps.className,
           'AMInstallButton-button--enable',
         );
-      } else if (addon.type === ADDON_TYPE_THEME) {
-        buttonProps['data-browsertheme'] = JSON.stringify(addon.themeData);
```

Comment:
It looks like we can also remove the `data-browsertheme` prop defined at https://github.com/mozilla/addons-frontend/blob/ec02b2f5004bfb12bde3d90b0564da66e4076f6d/src/core/components/AMInstallButton/index.js#L92

and it might also be worth removing the `data-browsertheme` prop from https://github.com/mozilla/addons-frontend/blob/ec02b2f5004bfb12bde3d90b0564da66e4076f6d/src/ui/components/Button/index.js#L37 as well.

Reply:
ah, good catch. Thanks!